chart/env/prod.yaml

```diff
@@ -537,3 +541,31 @@ workers:
       limits:
         cpu: 2
         memory: "1Gi"
+
+webhook:
+  # Number of uvicorn workers for running the application
+  # (2 x $num_cores) + 1
+  # https://docs.gunicorn.org/en/stable/design.html#how-many-workers
+  uvicornNumWorkers: "9"
+  nodeSelector:
+    role-datasets-server-webhook: "true"
+  tolerations:
+    - key: "huggingface.co/datasets-server-webhook"
+      operator: "Exists"
+      effect: "NoSchedule"
+  replicas: 20
+  service:
+    type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/target-node-labels: role-datasets-server-webhook=true
+      alb.ingress.kubernetes.io/actions.openapi-redirect: '{"Type":"redirect","RedirectConfig":{"Host":"raw.githubusercontent.com","Path":"/huggingface/dataset-viewer/main/docs/source/openapi.json","Port":"443","Protocol":"HTTPS","Query":"#{query}","StatusCode":"HTTP_302"}}'
+      alb.ingress.kubernetes.io/actions.metrics-unauthorized: '{"type":"fixed-response","fixedResponseConfig":{"contentType":"text/plain","statusCode":"401","messageBody":"401 Unauthorized"}}'
+  resources:
+    requests:
+      cpu: 4
+      memory: "14Gi"
```

Comment:
I don't think we need that quantity of RAM (hopefully)

Reply:
I put 4Gi, feel free to suggest another value if needed